modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -156,13 +156,26 @@ Limit the number of objects to retrieve through DataSense to a few objects using
 
 == Log Requests and Responses [[log-requests-responses]]
 
-To log requests and responses when using the connector, configure the `log4j.xml` configuration file for the Mule app as follows:
+To log requests and responses when using the connector, add the following line to the `Loggers` element of the `log4j2.xml` configuration file for the Mule app:
 
 [source,xml,linenums]
 ----
 <AsyncLogger name="org.mule.modules.siebel.internal.service"
    level="DEBUG"/>
 ----
+
+You can view the app log  as follows:
+
+* If you’re running the app from the Anypoint Platform, the output is visible in the Anypoint Studio console window.
+
+* If you’re using Mule from the command line to run the app, the app log is visible in your OS console.
+
+You can also view the application log in this default location, unless the log file path was customized in the app’s log file (`log4j2.xml`):
```

Comment:
Should use either all "application" or all "app" but not go back and forth.

Reply:
Agreed